### PR TITLE
[PD-949] Unsubscribe notifications for unsubscribing by email

### PR DESCRIPTION
**Pull request by aspidites**
2015-01-21-162804_1017x145_scrot

* Changed the wording slightly
* added the notifications when a user clicks unsubscribe (all) in an email
* updated opt out unit test
* wrote unsubscribe unit test
* send_opt_out_notifications is parameterized now (as well as pluralized)

### CODE REVIEW

File: templates/mysearches/email_opt_out_message.html [hunk 1]
```diff
@@ -1 +1 @@
-<p>{% if user.get_full_name %}{{ user.get_full_name }} ({{ user.email }}){% else %}{{ user.email }}{% endif %} from <a href="{% url 'partner_overview' %}?partner={{ partner.id }}">{{ partner.name }}</a> has opted out of receiving saved search emails.</p>
+<p>{% if user.get_full_name %}{{ user.get_full_name }} ({{ user.email }}){% else %}{{ user.email }}{% endif %} from <a href="{% url 'partner_overview' %}?partner={{ partner.id }}">{{ partner.name }}</a> has unsubscribed from one or more saved search emails.</p>
```
Comment (tdphillips) [inline templates/mysearches/email_opt_out_message.html]: Should we specify "one or more of [your|your company's] saved search emails"? I can see someone complaining about getting notifications and not knowing why.

Comment (aspidites) [inline templates/mysearches/email_opt_out_message.html]: Not sure. I suspect that the big red "unsubscribed" tag introduced by @galitskyd  would remove confusion. Also, given that this feature has existed for opted out notifications for a while and no one has complained, I don't now that it is likely.

Comment (aspidites) [inline templates/mysearches/email_opt_out_message.html]: Went ahead and added company info:
2015-01-21-165814_1006x159_scrot

**Comment by aspidites** [comment]
Reproducing this so it's not buried:

Went ahead and added company info:
2015-01-21-165814_1006x159_scrot

**Pull request merged**